bpf/lib/common.h

```diff
@@ -264,7 +264,11 @@ enum {
 #define NOTIFY_CAPTURE_HDR \
 	NOTIFY_COMMON_HDR						\
 	__u32		len_orig;	/* Length of original packet */	\
-	__u32		len_cap;	/* Length of captured bytes */
+	__u16		len_cap;	/* Length of captured bytes */	\
+	__u16		version;	/* Capture header version */
```

Comment:
I was originally considering to add a `hdr_len` field but a version is fine as well.

Reply:
See discussion with John here, I think it's actually a bit cleaner this way.

https://github.com/cilium/cilium/pull/9321#discussion_r335526797